tasks/populate-release-notes-images/populate-release-notes-images.yaml

```diff
@@ -44,11 +44,10 @@ spec:
         for ((i = 0; i < $NUM_COMPONENTS; i++))
         do
             component=$(jq -c --argjson i "$i" '.components[$i]' "${SNAPSHOT_FILE}")
-            name=$(jq -r '.name' <<< $component)
-            repo=$(jq -r '.repository' <<< $component)
-            deliveryRepo=$(translate-delivery-repo $repo | jq -r '.[] | select(.repo=="redhat.io") | .url')
-            tags=$(jq -c '.tags' <<< $component)
-            image=$(jq -r '.containerImage' <<< $component)
+            name=$(jq -r '.name' <<< "$component")
+            deliveryRepo=$(jq -e -r '."rh-registry-repo"' <<< "$component")
```

Comment:
Can you just write `jq -er '.rh-registry-repo'` instead of `jq -e -r '."rh-registry-repo"'`? It looks weird as is (unless checkton is making you do this)

Reply:
fixed, thanks!

Reply:
you still have `rh-registry-repo` in double quotes. Any reason?

Reply:
@johnbieren this is why:
```
jq '.rh-registry-repo' <<< '{"rh-registry-repo": "bla"}'
jq: error: registry/0 is not defined at <top-level>, line 1:
.rh-registry-repo
jq: error: repo/0 is not defined at <top-level>, line 1:
.rh-registry-repo
jq: 2 compile errors
```

vs.
```
jq '."rh-registry-repo"' <<< '{"rh-registry-repo": "bla"}'
"bla"
```

I'm sure you've hit that in the past :) `jq` requires you to quote key names that have `-` or anything special in them.

Reply:
Ah, I just probably haven't hit that lately enough. Thanks for showing it